File: CPP/Test/src/Staff.cpp
#include "Staff.h"


Staff::Staff()
{
    cout << "Staff created" << endl;
}

Staff::~Staff()
{
    cout << "Staff destroyed" << endl;
}

void  Staff::test()
 {
     cout << "Test" << endl;
 }

 void Staff::setName(string first_name,string last_name){
    m_first_name = first_name;
    m_last_name = last_name;
 }

 string Staff::getName(){

    return  "name : " + m_first_name +  " "  + m_last_name ;

 }

File: JAVA/EmployeeManagement/src/Staff.java

public abstract class Staff {
	String name;
	
	public abstract void calculateSalary();
	
	public Staff() {
		name="";
	}

	public Staff(String name) {
		super();
		this.name = name;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}
	
}
